common/src/main/proto/TransportMessages.proto

```diff
@@ -281,6 +281,7 @@ message PbHeartbeatFromApplication {
   int64 fileCount = 3 ;
   string requestId = 4;
   repeated PbWorkerInfo needCheckedWorkerList = 5;
+  bool reply = 6;
```

Comment:
shouldResponse?

Reply:
ok, it is renamed to shouldResponseMoreInfos

Reply:
I prefer to `shouldResponse`, `shouldResponseMoreInfos` is redundant

Reply:
ok, fixed